com.kms.cura.dal/src/com/kms/cura/dal/database/DoctorUserDatabaseHelper.java

```diff
@@ -532,5 +537,32 @@ public Entity updateDoctor(DoctorUserEntity doctorUserEntity) throws NumberForma
 			con.setAutoCommit(true);
 		}
 	}
+	
+	public Entity querybyID(int id) throws SQLException, ClassNotFoundException {
```

Comment:
@hieumai no i have checked and the id in the user table is different from the id in patient and doctor table